src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import Button from "../../inline/Button";
+
+import s from "./TwoParagraphBlock.module.css";
+
+/* PropType shapes */
+
+const ImagePropType = PropTypes.shape({
+  url: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
+});
+
+const TwoParagraphBlock = ({
+  title,
+  paragraph1,
+  paragraph2,
+  image,
+  ctaButton,
+}) => {
+  const GridAreaTopLeft = () => (
+    <div className={s.gridAreaTopLeft}>
+      <h1 className={s.title}>{title}</h1>
+    </div>
+  );
+
+  const GridAreaTopRight = () => (
+    <div className={s.gridAreaTopRight}>
+      <div className={s.paragraph1Wrapper}>{paragraph1}</div>
+    </div>
+  );
+
+  const GridAreaBottomLeft = () => (
+    <div className={s.gridAreaBottomLeft}>
+      <div className={s.paragraph2Wrapper}>{paragraph2}</div>
+      <div className={s.ctaButton}>
+        <Button
+          text={ctaButton.text}
+          internalLink={ctaButton.internalLink}
+          externalLink={ctaButton.externalLink}
+          onClick={ctaButton.onClick}
+        />
```

Comment:
A mapping behavior should be used since the Button class is expected a propType array

```suggestion
 {ctaButton.map((button) => (
          <div key={button.text}>
            <Button
              text={button.text}
              internalLink={button.internalLink}
              externalLink={button.externalLink}
              onClick={button.onClick}
            />
          </div>
        ))}
```